rest-service/manager_rest/security/audit.py

```diff
@@ -10,6 +11,8 @@ def set_audit_method(method: str):
 
 def set_username(username: str):
     g.audit_username = username
```

Comment:
Do we still need to set it in the flask session if it's in the DB session already? (I'm not sure where else that's used)

Reply:
That's what I get for reviewing this a commit at a time- I see you removed this on the next commit- ignore this!